cdap-app-fabric/src/test/java/co/cask/cdap/gateway/handlers/InMemoryAuthorizer.java

```diff
@@ -0,0 +1,82 @@
+/*
+ * Copyright © 2015-2016 Cask Data, Inc.
```

Comment:
I just moved it to a new location. not sure what the policy is. it was only used in tests, so I moved it to src/test...